README.md

```diff
@@ -1,39 +1,67 @@
+# deepin-anything
+
 Something like everything, but nothing is really like anything...
 
-anything的开发源自于Windows下的everything，它致力于为Linux用户提供一个闪电般速度的文件名搜索功能，并提供了离线搜索功能。原来在Linux中有一个类似的rlocate的程序，但是那个程序有点太老了，而重写一个更好的也不难，因此就重写了一个。
+The development of anything is derived from everything under Windows, and it is committed to providing a lightning-fast filename search function for Linux users, and provides offline search functions. Originally there was a similar rlocate program in Linux, but it was too old, so it was rewritten.
+
+### Build dependencies
+
+*The current development branch is **master**, build dependencies may change without updating this description, please refer to `./debian/control` for a list of build dependencies*.
+
+- debhelper (>= 9)
+- dkms
+- qtbase5-dev
+- pkg-config
+- libudisks2-qt5-dev
+- libmount-dev
+- libdtkcore-dev
+- libglib2.0-dev
+- libpcre3-dev
+
+## Installation
+
+### Build from source code
+
+1. Make sure all dependencies are installed.
 
-在处理器上，我们希望此工具能覆盖x86、龙芯与申威平台，当然，平台越多越好。
+   *Package names may be different for different distributions, if your distribution provides deepin-anything, please check the packaging scripts provided by your distribution.*
 
-具体使用测试请参见`docs/end_user_tester.md`，应用程序开发人员请参看`docs/app_developer.md`，库开发人员请参看`docs/lib_developer.md`，有关设计方面的考虑请参看`docs/design_considerations.md`。
+If you are using [Deepin](https://distrowatch.com/table.php?distribution=deepin) or any other Debian based distribution that provides deepin-anything：
 
-# 编译
+```shell
+# apt build-dep deepin-anything
+```
 
-进入根目录，直接`make`即可编译生成对应的内核模块与用户态程序。编译所有程序需要安装`make`与`gcc`，编译内核模块需要安装内核头文件，编译用户态程序没有额外的依赖。编译生成的内核模块名为`vfs_monitor.ko`(使用了`-O3`参数优化)，在`kernelmod`目录下。编译生成的用户态共享库位于`library/bin`目录，分为调试版(`debug/libanything.so`)与发布版(`release/libanything.so`)，编译生成的用户态命令行测试程序位于`cli/bin`目录，分为调试版(`debug/anything_cli`)与发布版(`release/anything_cli`)。
+2. Build：
 
-其中内核模块位于`kernelmod`目录，用户态共享库位于`library`目录，命令行测试程序位于`cli`目录。当前，用户态程序支持所有平台，内核模块暂时仅支持x86与龙芯平台。
+```shell
+$ cd deepin-anything
+$ make
+```
 
-当然，你也可以进入每个目录执行`make`来分模块构建，但是因为`cli`里的测试程序依赖于`library`里的动态库，所以需要首先构建`library`，再构建`cli`。
+3. Installation：
 
-在运行环境上，内核模块需要依赖内核的CONFIG\_KPROBES选项，用户态程序仅依赖于glibc库。
+```shell
+# make install
+```
 
-# 对比
+## Documentations
 
-为了说明anything的特性，先来做一个简单但仍有典型意义的文件搜索对比测试。
+## Getting help
 
-首先说明下测试环境，测试环境物理机为ThinkPad x230，8G内存，机械硬盘。虚拟机为运行在VirtualBox里的Deepin 15.4，内存为2G，处理器为单核2.9 GHz。
+- [Official Forum](https://bbs.deepin.org/) for generic discussion and help.
+- [Developer Center](https://github.com/linuxdeepin/developer-center) for BUG report and suggestions.
+- [Wiki](https://wiki.deepin.org/)
+- [docs](https://github.com/wangrong1069/deepin-anything/tree/master/docs)
```

Comment:
does this docs should be the up steam link, right?

Reply:
yes, it is mistake.